helper-npm-install-peer-deps

```diff
@@ -34,7 +34,7 @@ if [ -e package.json ]; then
 
     npm ls --production --parseable 2>&1 >/dev/null | \
       sed -n -e 's/^npm ERR! peer dep missing: \(.*\),.*/\1/p' | \
-        xargs -I{} echo -n '"{}" ' | \
-          xargs -I{} npm install --no-package-lock --no-save "{}"
+          xargs -I{} echo -n '"{}" ' | \
+            xargs npm install --no-package-lock --no-save
```

Comment:
Looks like it was previously failing for your repo because the default `replsize` of 255 bytes assumed when no `-S` flag is passed to xargs was too small for all the peer dependencies required (!) Was there a reason you used the `-I replstr` templating for the command in #38, @JakeChampion? (I know very little about the idiosyncrasies of xargs...) If there was it might be required to specify a larger – but still finite – `replsize` value via `-S` here, but hopefully that won't be necessary.
```suggestion
            xargs -S16383 npm install --no-package-lock --no-save
```